docs/content/concepts/layer/control/catalog/sp800-53rev5-example.md

```diff
@@ -0,0 +1,340 @@
+
+Since OSCAL is designed purposefully to represent such data sets (in a specific semantic domain, namely the [NIST Risk Management Framework](https://csrc.nist.gov/publications/detail/sp/800-37/rev-2/final) or RMF), the casting of this information into OSCAL encoding can be straightforward, economical and clean.
+
+## Control, ID and title
+
+This control has a number of high-level data elements, including a security control identifier ("AC-1"), a title ("POLICY AND PROCEDURES"), the control (text) itself, supplemental guidance, control enhancements (although not in this example), and references.
+
+See the source code as currently available in [raw XML from Github](https://raw.githubusercontent.com/usnistgov/oscal-content/main/nist.gov/SP800-53/rev5/xml/NIST_SP-800-53_rev5_catalog.xml). The control is identified in code with tagging that looks *mainly* as follows, with slight simplifactions. Control AC-1 starts:
+
+{{<highlight xml "nowrap=false" >}}
+<control class="SP800-53" id="ac-1">
+    <title>Policy and Procedures</title>
+{{< / highlight >}}
+
+- The control is assigned a class, "SP800-53". This serves as an indicator to a downstream processor of the control's origin (with respect to the catalog or catalog type), with implications for related expectations, including expectations in detail regarding how the control and its contents may be structured.
+- The control ID is "ac-1". All `id` values are unique within the document and serve for addressing and linking. The lexical form of the `id` is designed to be consistent with its formal (conventional) code or label, which is *also* encoded within the control (in this case, "AC-1"). While such regularities are normal and to be expected, this one is a feature of this data set, not a requirement of OSCAL generally.
+- The `<title>` of the control captures the text of the control's title as given in display and navigation. Unlike `id` values, titles are not required to be unique, and indeed in this catalog, each family of controls is headed by a control entitled "Policy and Procedures". In an application, the control title can be adjusted for display in any way needed - for example, prepended with the name of the family as **Access Control - Policy and Procedures", or converted into all upper case.
+
+## Parameters
+
+The control continues with `<param>` elements. This is the first place where the electronic file definitively and explicitly captures information about the source data that is left implicit in in its published form (that is, as a document). In this case, the information captured regards options for control configuration (by users of the catalog) in detail, which will subsequently be reflected in the control text as rendered, but which are called out here as first-class objects so that OSCAL consumers can handle them in appropriate ways.
```

Comment:
```suggestion
The control continues with `<param>` elements. This is the first place where the structured form captures information about the source data that is left implicit in in its published form (that is, as a document). In this case, the information captured regards options for control configuration (by users of the catalog) in detail, which will subsequently be reflected in the control text as rendered, but which are called out here as first-class objects so that OSCAL consumers can handle them in appropriate ways.
```